build fails with Gradle 5+

**Describe the bug**
```
$ gradle tasks

FAILURE: Build failed with an exception.

* Where:
Build file '/root/Java-WebSocket/build.gradle' line: 45

* What went wrong:
A problem occurred evaluating root project 'Java-WebSocket'.
> Cannot add task 'wrapper' as a task with that name already exists.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
(error 1)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the project
2. run `gradle tasks` with Gradle version 5+

**Additional context**
The issue is described in this stackoverflow question: https://stackoverflow.com/questions/53709282/cannot-add-task-wrapper-as-a-task-with-that-name-already-exists

--- a/build.gradle
+++ b/build.gradle
@@ -41,8 +41,3 @@ dependencies {
 //		}
 //    }
 //}
-
-task wrapper(type: Wrapper) {
-    gradleVersion = '1.4'
-}
-

